src/Sentry/SentryLog.cs

```diff
@@ -0,0 +1,281 @@
+using Sentry.Extensibility;
+using Sentry.Infrastructure;
+using Sentry.Internal;
+using Sentry.Protocol;
+
+namespace Sentry;
+
+/// <summary>
+/// Represents the Sentry Log protocol.
```

Comment:
Is there any reason this isn't in the `Sentry.Protocol` namespace?

Reply:
`SentryLog` is used in the `SetBeforeSendLog`-provided callback.
Should user code prefer explicit typing, then `Sentry.Protocol` would need to be included.

Alternatively, we could maybe split these, in a `SentryLog` type which is user facing and basically a user-editable property bag, which is further down the pipeline used to create the Envelope and Envelope-Item via a type that is in `Sentry.Protocol`.

What do you think is best?